docs/docs/how-tos/develop-local-packages.md

```diff
@@ -0,0 +1,66 @@
+---
+id: develop-local-packages
+title: Develop Local Packages on Nebari
+description: How to install a dev package from a local project
+---
+
+# How to Develop Local Packages on Nebari
+
+As a software development platform, Nebari can be used for active package development. Since `conda-store` manages all environments, a dynamic installation of a local package requires additional care (after all, `pip install -e .` is explicitly disallowed for various reasons).
+
+While the setup for developing local packages may require unfamiliar extra steps for new users, the total time for setup is minimal.
+
+## Why can't I just `pip install -e` my package?
+
+Nebari prohibits the standard local installation of packages, e.g. `pip install -e .`. On your personal computer, when you `pip install` a package into a conda environment, it will be placed inside of the conda environment folder structure. However, on Nebari, the environments are managed by `conda-store` in a directory structure which is read-only to users.
```

Comment:
```suggestion
Nebari blocks the standard local installation of packages, e.g. `pip install -e .`. On your personal computer, when you `pip install` a package into a conda environment, it will be placed inside of the conda environment folder structure. However, on Nebari, the environments are managed by `conda-store` in a directory structure which is read-only to users.
```